webui/react/src/dev.ts

```diff
@@ -0,0 +1,18 @@
+/* Tools and tweaks for dev environments */
+
+import * as Api from 'services/api';
+import { globalStorage } from 'utils/storage';
+
+const setServerAddress = (address: string) => {
+  globalStorage.set('server-address', address);
+  Api.updateDetApi({ basePath: address });
+};
+
+window.dev = window.dev || {};
```

Comment:
**question:** should this just be?
```
window.dev = {
  setServerAddress,
  resetServerAddress: () => globalStorage.remove('server-address'),
};
```
In the case the `window.dev` is set, it's probably not by us and the question is would we want to leave it alone? Most likely we'll just choose a different namespace if they conflicted.

Reply:
I'm not sure which one's better. the way it is would cause less conflicts but might mask some issues?